src/org/opendatakit/briefcase/ui/pull/PullPanel.java

```diff
@@ -63,11 +63,15 @@ public PullPanel(TransferPanelForm view, TransferForms forms, BriefcasePreferenc
     getContainer().addComponentListener(analytics.buildComponentListener("Pull"));
 
     // Read prefs and load saved remote server if available
-    this.source = RemoteServer.readPreferences(tabPreferences).flatMap(view::preloadSource);
-    this.source.ifPresent(source -> {
-      forms.load(source.getFormList());
-      view.refresh();
-      updateActionButtons();
+    source = RemoteServer.readPreferences(tabPreferences).flatMap(view::preloadSource);
+    source.ifPresent(source -> {
+      try {
+        forms.load(source.getFormList());
+        view.refresh();
+        updateActionButtons();
+      } catch (Exception e) {
+        System.err.println(e.toString());
```

Comment:
Since this code is only executed when we're in UI mode, log this error instead of using `System.err`.

Also, the side effects of this should be exactly the same as the callback for `view.onReset()` in line 87 (we want to forget about that server conf and reset the UI). Maybe extract to a method and reuse in both places?

Reply:
Are you sure you want to discard the server configuration? What if it’s a temporary condition?

Reply:
Changing to logging:
`2018-08-12 18:57:18,570 [main] WARN  o.o.briefcase.ui.pull.PullPanel - Unable to get form list from http://localhost:8080: org.opendatakit.briefcase.reused.http.HttpException: Connection refused`